nav_msgs/OccupancyGrid o
int64 dr
int64 dx
int64 maxRadius
---
geometry_msgs/Point p
